tests/homePage/HomePage.test.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect } from '@playwright/test';
+
+/**
+ * Test that logo shows up
+ */
+
+test('Should display Logo', async ({ page }) => {
+    await page.goto("https://techstartucalgary.com");
+    // Find the logo image by class
+    const logo = page.locator('img.homePage__logo');
+    await expect(logo).toBeVisible();
+    await page.close();
+
+});
+
+
+/**
+ * Test join team button
+ */
+test('join button should navigate to Apply page', async function ({ page }) {
+    await page.goto("https://techstartucalgary.com");
+
+    // find the join team button
+    await page.getByRole('link', { name: 'theTeam.join()' }).click();
```

Comment:
This test is failing for firefox, I think its because firefox is more strict with text matching. For flexibility you can use regex, something like this - `await page.getByRole('link', { name: /theTeam\.join\(\)/ }).click();`

Reply:
Thanks. I've made the change